src/lib/adapter.ts

```diff
@@ -148,6 +149,23 @@ class GleeAdapter extends EventEmitter {
     return this._serverUrlExpanded
   }
 
+  async resolveConfig(protocol: string) {
```

Comment:
Would it make sense to call it `resolveProtocolConfig` instead?

```suggestion
  async resolveProtocolConfig(protocol: string) {
```